### DISCUSSION TITLE
Current ToDo's

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/reactsocialnetwork#160] [General]
We still have a lot of the app to create.

-- Components/Panels
User Profile component
Form for creating User Profile
Settings component (General /Global app settings)
User Dashboard (User stats)
Group Dashboard (Group statistics, moderating group)
Admin/Moderator Dashboard
Post editor component (Use Lexical or?)
Chat widget (bring in a live chat API via cdn??)
Message widget or component
Friends component (Shows friends list, suggested, mutual friends, etc)
Notification widget (off the bell icon, shows new notifications)

Also should we include:
OpenGraph meta tags
Twitter meta tags
Fallback fonts from Google Fonts API
RSS feed?
Google Tag Manager
Google Analytics
Google Ads

**Comment by gbowne1** [comment]
@manuel12 @balajik @pawel975

**Comment by manuel12** [comment]
@gbowne1 I would like to work next  on:

1. User Profile component: created #297
2. Friends component: 
    - Finishing backend routes and database
    - Adding API tests
    - Creating Friends component UI 
3. Also would be good if  we created documentation on Coding Style Guide(How to name functions and variables and how to document components or util functions) and on Naming Conventions(How to name files, components, routes, tests, etc)

This last point about documentation can save a lot of time for newcomers that would otherwise be spent wondering on how to name things. I've saved time by creating documents like this even on personal projects where I am the only dev! :)

**Reply by gbowne1** [reply]
That sounds good to me. 

I agree.  You might work with the person I suggested on Slack to get the documentation done.  I can assign it to them.

We had the same issue over on CodeBooker repo.  People didn't know how to set up the project.

I am hoping @balajik can help us too.  

I have some sample code that might help enhance Profile I can share.

@manuel12
